sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -1,19 +1,70 @@
 ﻿# Contributing 
 
-Coming Soon...
+Thank you for your interest in contributing to the Event Hubs client library.  As an open source effort, we're excited to welcome feedback and contributions from the community.  A great first step in sharing your thoughts and understanding where help is needed would be to take a look at the [open issues](https://github.com/Azure/azure-sdk-for-net/issues?q=is%3Aopen+is%3Aissue+label%3AClient+label%3A%22Event+Hubs%22).
```

Comment:
I'm not familiar with track1 vs track2 for Event Hubs, but will the open issues link shows only the track2 issues?

Reply:
There's no way to distinguish, unfortunately;  they share a set of labels.   This just narrows the list by the label  "Client" and "Event Hubs".

Reply:
Are the projects in different folders? Maybe you can add that so people now where to look

Reply:
That's a darn good idea.

Reply:
Looking into it, I think that makes sense to apply to issues as we triage.  Anyone reading this contributing guide is already in the containing folder for the track two code... but having some way to differentiate at the issue level would be really helpful.

@AlexGhiondea:  Any thoughts here?  Should this be some kind of label that we can use?   Track One / Track Two sounds bad, but maybe there's something that is as expressive and less offensive?

Reply:
This is a good point -- right now we don't have a way of distinguishing between track 1 and track 2... We need to figure out a way to distinguish those.